include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1424,24 +1424,42 @@ int bt_le_ext_adv_create(const struct bt_le_adv_param *param,
 
 struct bt_le_ext_adv_start_param {
 	/**
-	 * @brief Advertiser timeout (N * 10 ms).
+	 * @brief Maximum advertising set duration (N * 10 ms)
+	 *
+	 * The advertising set can be automatically disabled after a certain
+	 * amount of time has passed since it first appeared on air.
 	 *
-	 * Application will be notified by the advertiser sent callback.
-	 * Set to zero for no timeout.
+	 * Set to zero for no limit. Set in units of 10ms.
+	 *
+	 * When the advertising set is automatically disabled because of this
+	 * limit, @ref bt_le_ext_adv_cb.sent will fire.
```

Comment:
We don't use 'fire' phrase when referring to act of callback getting called by the system. In this file looks like established pattern is to use phrase - 'will be called'

```suggestion
	 * limit, @ref bt_le_ext_adv_cb.sent callback will be called.
```

Reply:
I like Omkar's suggestion, but perhaps it should just be 
```c
limit, @ref bt_le_ext_adv_cb.sent will be called.
```

Seems superfluous to add the `callback`